documentation/api/definitions.md

```diff
@@ -1,5 +1,15 @@
 # Definitions
 
+## DotnetMonitorInfo
+
+Object describing diagnostic/automation information about the executing instance of Dotnet-Monitor.
```

Comment:
```suggestion
Object describing diagnostic/automation information about the executing instance of `dotnet monitor`.
```

We try to stylize it as `dotnet monitor` in the documentation in order to promote the usage of launching it through the `dotnet` executable.